src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
```

Comment:
Is this field (Namespace) used in the data uploaded to BenchView? If not, then we can remove it.

---
In reply to: [107045986](https://github.com/Microsoft/xunit-performance/pull/208#discussion_r107045986) [](ancestors = 107045986)